scripts/Muon/GUI/Common/utilities/algorithm_utils.py

```diff
@@ -94,6 +94,42 @@ def run_PhaseQuad(parameters_dict, alg):
     return alg.getProperty("OutputWorkspace").value
 
 
+def run_PaddingAndApodization(parameters_dict):
```

Comment:
at some point I think it will be worth either refactoring the frequency specific algs out into a different file or placing them at the bottom of this one with a comment explaining the below are frequency specific